dev/io.openliberty.microprofile.health.4.0.internal_fat/build.gradle

```diff
@@ -0,0 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+addRequiredLibraries.dependsOn addJakartaTransformer
```

Comment:
There is no application, so not sure why this file is needed.  Jakarta transformer isn't used.